### DISCUSSION TITLE
Running as a docker container not cleaning up old files

### DISCUSSION THREAD

**Discussion by dkarzon** [ep1cman/unifi-protect-backup#71] [General]
I am running this as a docker container on an unraid instance syncing from a UDM Pro to OneDrive. The sync up to OneDrive works great. However, I have noticed it doesn't seem to clean up the old files (those that fall outside of the retention period) when the container is left constantly running.
As soon as I restart the container it then cleans up the old files as of the current time but continues to run only uploading.

Is this a known issue with the docker container method of running it or is there something I am missing to have the clean up functionality on an always on docker container?

**Comment by ep1cman** [comment]
This is definitely an issue, I have opened one here: https://github.com/ep1cman/unifi-protect-backup/issues/72 please discuss futher there
